docs/reference/cache.md

```diff
@@ -1,36 +1,56 @@
+## Memory Caches
 
-## In Memory Caches
+Ratify provides a unified cache API for cache read/write. Caching providers implement the API and are registered on ratify creation. Users can configure which cache provider to use via helm chart/cli. Here are the current cche providers supported:
 
-There are four different in-memory caches used for separate purposes. There are three different cache implementations as well.
+### Ristretto (default)
 
-### K8s API Machinery LRU Util Cache
+Ristretto is a highly performant in-memory caching library. The ristretto cache is configured as LRU.
+See [docs](https://github.com/dgraph-io/ristretto).
 
-The util cache, implemented in k8s apimachinery package, is an LRU in memory cache. See [docs](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/cache#LRUExpireCache). It is used as the http server verification handler cache. This cache acts as a short lived response cache. In particular, this cache deduplicates redundant cache requests received by Ratify at once.
+When to use Ristretto as the cache provider?
+- using the ratify quick start
+- only using ratify with notary verifier configured
+- using ratify as a single pod
 
-### Sync Map
+### Dapr
 
-This is a thread-safe mutex-lock map defined in the default go `sync` package. This is a very rudimentary data store that we use as a cache. See [docs](https://pkg.go.dev/sync#Map).
+Dapr (Distributed Application Runtime) supports many different state-stores both open source and cloud specific. Redis is the prefferred state-store implementation. Redis is a memory-based key-value distributed data-store. It is atomic and highly concurrent making it a good fit for distributed applications with multiple readers and writers. Ratify uses Redis as a centralized cache shared across multiple processes (each external plugin is run as a separate process) and across mutliple ratify replicas.
 
-The authentication cache for ORAS is backed by a sync map. This cache is a key value mapping between the fully qualified subject reference string (hostname, repository, and tag) and an ORAS repository client initialized with credentials fetched from the Authentication Provider specified. 
+Ratify REQUIRES that each Dapr state store be configured with a primary encryption key in order to guarantee confidentiality of the cached data at rest. Please see guide [here](../examples/ratify-high-availability.md) on installing Ratify with Dapr Redis integration.
 
-The subject descriptor cache for ORAS also uses a sync map. It maps a digest based full qualifed subject image reference to a resolved subject descriptor. This is used by the `GetSubjectDescriptor` ORAS store implementation. The `/mutate` path heavily relies on this cache due to the often redundant mutate requests Ratify gets during K8s resource creation.
+The Ratify chart does NOT come with built-in installation support for Dapr or Redis. User's must manage and configure on their own.
```

Comment:
In the [guide](../examples/ratify-high-availability.md) , i see Redis/Dapr install steps, how does the customer manage the manage Dapr? does ratify run it for them if feature flag is enabled?